sql/core/src/main/scala/org/apache/spark/sql/execution/WholeStageCodegen.scala

```diff
@@ -305,7 +307,7 @@ case class WholeStageCodegen(child: SparkPlan) extends UnaryNode with CodegenSup
    * @return the tuple of the codegen context and the actual generated source.
    */
   def doCodeGen(): (CodegenContext, String) = {
-    val ctx = new CodegenContext
+    val ctx = new CodegenContext(SQLContext.getActive().get.conf)
```

Comment:
Oh, I haven't test it non-local mode. If then, is there any way to access the configuration of SQLContext in executors?